tests/Config/ConfigurationLoaderTest.php

```diff
@@ -89,6 +89,70 @@ public function testLoadsConfiguration(): void
         $this->assertEquals('cold_start_bandit', $bandit->banditKey);
     }
 
+
+    public function testSetsConfigurationTimestamp(): void
+    {
+        // Load mock response data
+        $flagsRaw = file_get_contents(self::MOCK_RESPONSE_FILENAME);
+        $flagsResourceResponse = new APIResource(
+            $flagsRaw,
+            true,
+            "ETAG"
+        );
+        $banditsRaw = '{
```

Comment:
Are bandits necessary for testing the cache? Or is it just to make the test more comprehensive?

Reply:
you're right. the bandit payload is superfluous here and not used by the loader/tested here.